shotover-proxy/src/frame/cassandra.rs

```diff
@@ -60,8 +63,8 @@ pub(crate) fn metadata(bytes: &[u8]) -> Result<CassandraMetadata> {
     })
 }
 
-/// Count queries only from an unparsed Cassandra frame
-pub(crate) fn get_message_count(bytes: &[u8]) -> Result<NonZeroU32> {
+/// Count "cells" only from an unparsed Cassandra frame
+pub(crate) fn cell_count(bytes: &[u8]) -> Result<NonZeroU32> {
```

Comment:
Looks like you messed up a merge or something?

Reply:
No, I just branched off #574 so I can add onto the existing batch statement tests.

Reply:
oh huh, I forgot that was still living in another PR.
ben plz